test/src/evaluation.spec.ts

```diff
@@ -48,31 +48,31 @@ describe('Evaluation specs', function () {
     it('should transfer NaN', async () => {
       const {page} = getTestState();
 
-      const result = await page.evaluate(a => {
```

Comment:
why are the exact types required?

Reply:
I was getting some error. I tried removing them now and it seems to work. I'm happy to revert this change.

Reply:
yes, let's revert (we should not need any changes to test files)